### ISSUE TITLE
'Against' check issues

### ISSUE THREAD

**Issue by Lysandr0** [CaptainObvious0/NoCheatPlus#32]
### Full output of /ncp version
```
[INFO] ---- Version information ----
[INFO] #### Server ####
[INFO] git-Spigot-e6eb36f-d3ed151 ~MC: 1.13.2~
[INFO] detected: 1.13.2
[INFO] #### NoCheatPlus ####
[INFO] Plugin: 3.16.1-SNAPSHOT-CustomFork-v1.3-beta.3
[INFO] MCAccess: 1.4.5-1.12.2|? / CB-Reflect
[INFO] Features:
[INFO] blocks: BlocksMC1_4 | BlocksMC1_5 | BlocksMC1_6_1 | BlocksMC1_7_2 | BlocksMC1_8 | BlocksMC1_9 | BlocksMC1_10 | BlocksMC1_11 | BlocksMC1_12 | BlocksMC1_13
[INFO] checks: FastConsume | Gutenberg | HotFixFallingBlockPortalEnter | AttackFrequency | FlyingFrequency | KeepAliveFrequency
[INFO] defaults: pvpKnockBackVelocity
[INFO] packet-listeners: UseEntityAdapter | MovingFlying | OutgoingPosition | KeepAliveAdapter | SoundDistance | DebugAdapter
[INFO] Hooks: AllViolations~NCP~ 1.0
[INFO] #### Related Plugins ####
[INFO] ProtocolLib v4.4.0 | ProtocolSupport v4.29-dev
```

### Describe the issue
1) The check doesn’t seem to be working anymore (tried to use some liquids hacks and it did not prevent it). No errors are shown in the console or anything at all for that matter.
This is a fork issue.
2) Taken from NoCheatPlus/Issues: [#578](https://github.com/NoCheatPlus/Issues/issues/578)

### Extra links/Videos (Including debug logs)
https://gyazo.com/97be9f42cffbf26a56e3fd2aa9ebfd4f

**Comment by CaptainObvious0** [comment]
The issue is somewhat contradicted. You point out that the check is not working at all, where as the ticket you linked says that they are experiencing false positives with the check. 

Will be checked out later.

**Comment by Lysandr0** [comment]
Originally, I just wanted to take the ticket from NCP’s issues tracker and post it here, then when I was toying around with a hacked client to test stuff out, I happened to discover that the check was not working... :p
(At least for me), so I just merged both issues in one ticket

**Comment by CaptainObvious0** [comment]
I believe that this issue is caused by ProtocolSupport, but I can't confirm because I don't have a client that has a liquid interact mod on 1.13. 

I was not able to reproduce any false positives from the check.

**Comment by Lysandr0** [comment]
That's odd... I was able to reproduce it even without ProtocolSupport (as shown in the /ncp ver's ticket) back then... (Side note: it was a 1.12.2 server)
Perhaps this got unintentionally fixed while updating NCP to 1.13 (?) (Thinking about all the block shape/models updates etc...) :p

I'll have to verify with the latest vanilla NCP build and without PS some time.

_Edit:_
PS + NCP (vanilla) b.1149 (+ 1.12 cheat client) (+1.13 server) = check not working
NCP (vanilla) b.1153 only (+1.13 cheat client) (+1.13 server)  = Same result

Seems to be an issue with NoCheatPlus + a 1.13 server, since the check was working correctly on a 1.12 server, regardless the presence of PS.

**Comment by Lysandr0** [comment]
Fixed through @Xaw3ep's fork.
